x-pack/plugins/actions/server/actions_client.test.ts

```diff
@@ -1186,6 +1259,7 @@ describe('getAll()', () => {
         aggregations: {
           '1': { doc_count: 6 },
           testPreconfigured: { doc_count: 2 },
+          'system-connector-.cases': { doc_count: 2 },
```

Comment:
@pmuellr @ymao1  Is this assumption correct? Will the aggregation always return an entry for the in-memory connectors?

Reply:
Yes, that's correct. Looking at the code, it combines the saved object connectors with the preconfigured (in-memory) connectors and then adds each connector to the aggregation to see how many rules are using each. So even if no rules are using the in-memory connector, a result should be returned.

Reply:
Thanks!